package/yast2-packager.spec

```diff
@@ -101,6 +105,8 @@ This package contains the libraries and modules for software management.
 
 %install
 %yast_install
+mkdir -p %{buildroot}%{_bindir}
```

Comment:
Actually why the wrapper script cannot be part of the tarball and installed the usual way? With this approach the wrapper will not be installed when you run `rake install` from the sources. That will be confusing, if you update the file locally your changes will not take effect...